cookiecutter.json

```diff
@@ -1,6 +1,7 @@
 {
     "formal_name": "App Name",
     "app_name": "{{ cookiecutter.formal_name|lower|replace(' ', '-') }}",
+    "class_name": "{{ cookiecutter.formal_name.title().replace(' ','').replace('-','').replace('!','').replace('.','').replace(',','') }}",
```

Comment:
What's the reason for adding `class_name`? If we need something that's a valid identifier, don't we already have that in the Briefcase `app_name` (whatever comes after `tool.briefcase.app` in the pyproject.toml file)?

Reply:
So this change isn't strictly necessary; it's been done for consistency with what Xcode does with a fresh project. The older `app_name` version is entirely viable; but if you add new modules to an Xcode project, you get `ClassName`-style names, not `app_name`.

It's entirely cosmetic, and not required at all; it was originally done because when we needed a separate folder for the Test class, it looked really odd to have `my_app` and `MyAppTest` in the same folder. Even though we removed that folder, I figured since the changes had been made and it would be less surprising to potential Xcode users, it was worthwhile keeping.